INDEX

```diff
@@ -219,5 +219,13 @@
         "description": "Irssi is a modular IRC client that currently has only text mode user interface, but 80-90% of the code isn't text mode specific, so other UIs could be created pretty easily.",
         "license": "https://raw.githubusercontent.com/freenas/iocage-plugin-qbittorrent/master/irssi-license",
         "official": true
+  },
+ "mineos": {
+        "MANIFEST": "mineos.json",
+        "name": "MineOS",
+        "primary_pkg": "https://github.com/hexparrot/mineos-node",
```

Comment:
He's grabbing the master branch, so make this look like the sickrage entry for `primary_pkg`